recipes-app/optee-client/files/control.tmpl

```diff
@@ -0,0 +1,51 @@
+Source: optee-client
```

Comment:
Hm, have you reported the issues here to [upstream Debian package](https://packages.debian.org/bookworm/optee-client-dev)? Seems that with this fixed there's no need to maintain this here, right?

Reply:
Do you mean the 0002.patch? I've upstreamed this patch to optee project , see https://github.com/OP-TEE/optee_client/commit/dd2d39b49975d2ada7870fe2b7f5a84d0d3860dc.

The target would be to use the upstream debian package provided by bookworm when debian 12 is released. However, currently there are still some todos, for example currently the debian upstream version use the RPMB emulation but not the real RPMB device, (see the RPMB_EMU=0 in the file `rules`), this might need introduce changes into linaro upstream for making it configurable via parameters. So at least for now, the debian upstream version could not be used directly.